### PR TITLE
Add Account Name Update Functionality

### PR DESCRIPTION
**Pull request by Publicker**
Once upon a time... in a realm not so far from this very codebase, a ticket arose. Its humble request? A mere account name update. But wow, it took longer than I thought (sorry about that!). Good news is, it's done now and you can change account names!

_(I hope this story makes up for the wait a little bit 😅)_

I've added a video to show how it works:

https://github.com/nofrixion/nofrixion.business/assets/52673485/eee71f46-b78e-489a-b942-58c3c7729c5f

### CODE REVIEW

File: packages/clients/src/clients/AccountsClient.ts [hunk 1]
```diff
@@ -76,4 +76,18 @@ export class AccountsClient extends BaseApiClient {
       url,
     )
   }
+
+  /**
+   * TODO: Updates an Account
```
Comment (donalnofrixion) [inline packages/clients/src/clients/AccountsClient.ts:81]: Remove the TODO?

**Review by donalnofrixion — APPROVED** [review]

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/73.

File: packages/clients/src/clients/AccountsClient.ts (hunk 1 above)
Comment (Publicker) [reply packages/clients/src/clients/AccountsClient.ts]: Oopsy! Thanks

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/73.

**Review by sauravmaiti22 — APPROVED** [review]

**Review by pabloi-nf — CHANGES_REQUESTED** [review]
We need to allow the user to discard the changes. This should happen if the user clicks outside the title or presses the ESC key

**Comment by donalnofrixion** [comment]
> We need to allow the user to discard the changes. This should happen if the user clicks outside the title or presses the ESC key

@Publicker This is implemented here
https://github.com/nofrixion/nofrixion.business/blob/develop/packages/components/src/components/ui/Tags/AddTag/AddTag.tsx

**Comment by Publicker** [comment]
> Looks good @Publicker
> 
> One question. I feel like clicking the edit icon should enter edit mode

@donalnofrixion Good idea! Done!

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/73.

**Review by donalnofrixion — APPROVED** [review]

**Comment by donalnofrixion** [comment]
🔥

**Review by arifmatin — APPROVED** [review]

**Pull request merged**